README.md

```diff
@@ -3224,6 +3229,425 @@ port-channel hash input offset. Valid values are integer or 'default'.
+| `match_length`                           | Not supported on N3k,N9k-F,N9k  |
+| `match_mac_list`                         | Not supported on N3k,N9k-F,N9k  |
+| `match_metric`                           | Not supported on N9k-F          |
+| `match_ospf_area`                        | Not supported on N3k,N5k,N6k,N9k-F,N9k <br> Supported in OS Version I5 and later on N3k,N9k  |
+| `match_vlan`                             | Not supported on N3k,N9k-F,N9k  |
+| `set_extcommunity_4bytes_additive`       | Not supported on N9k-F          |
+| `set_extcommunity_4bytes_non_transitive` | Not supported on N9k-F          |
+| `set_extcommunity_4bytes_transitive`     | Not supported on N9k-F          |
+| `set_extcommunity_cost_igp`              | Not supported on N9k-F          |
+| `set_extcommunity_cost_pre_bestpath`     | Not supported on N9k-F          |
+| `set_extcommunity_rt_additive`           | Not supported on N9k-F          |
+| `set_extcommunity_rt_asn`                | Not supported on N9k-F <br> Supported in OS Version I4 and before on N9k |
+| `set_forwarding_addr`                    | Not supported on N9k-F          |
+| `set_ipv4_default_next_hop`              | Not supported on N5k,N6k,N9k-F,N9k  |
+| `set_ipv4_default_next_hop_load_share`   | Not supported on N5k,N6k,N9k-F,N9k  |
+| `set_ipv4_next_hop`                      | Not supported on N9k-F          |
+| `set_ipv4_next_hop_load_share`           | Not supported on N5k,N6k,N9k-F <br> Supported in OS Version I5 and later on N9k  |
+| `set_ipv4_next_hop_redist`               | Supported on N5k,N6k,N7k,N9k-F <br> Supported in OS Version I5 and later on N3k,N9k  |
+| `set_ipv4_precedence`                    | Not supported on N9k-F          |
+| `set_ipv4_prefix`                        | Not supported on N5k,N6k,N9k-F  |
+OS Version I5 and later on N3k,N9k  |
+| `set_ipv6_default_next_hop`              | Not supported on N5k,N6k,N9k-F,N9k  |
+| `set_ipv6_default_next_hop_load_share`   | Not supported on N5k,N6k,N9k-F,N9k  |
+| `set_ipv6_next_hop`                      | Not supported on N9k-F          |
+| `set_ipv6_next_hop_load_share`           | Not supported on N5k,N6k,N9k-F <br> Supported in OS Version I5 and later on N9k  |
+| `set_ipv6_next_hop_redist`               | Supported on N5k,N6k,N7k,N9k-F <br> Supported in OS Version I5 and later on N3k,N9k  |
+| `set_ipv6_prefix`                        | Not supported on N5k,N6k,N9k-F  |
+| `set_vrf`                                | Supported on N7k                |
+
+#### Parameters
+
+##### Basic interface config attributes
+
+###### `ensure`
+Determine whether the route map config should be present or not. Valid values
+are 'present' and 'absent'.
+
+
+##### `description`
+Description of the route-map. Valid values are string, or keyword 'default'
+
+##### `match_as_number`
+Match BGP peer AS number. Valid values are an array of ranges or keyword 'default'

```

Comment:
I feel like this needs more explanation by way of example.  Maybe just a simple inline example with a description of what it means.